src/Microsoft.SqlTools.ServiceLayer/TableDesigner/Contracts/Requests/GeneratePreviewReportRequest.cs

```diff
@@ -11,7 +11,11 @@ namespace Microsoft.SqlTools.ServiceLayer.TableDesigner.Contracts
     /// The service request to generate preview report describing the changes.
     /// </summary>
 
+<<<<<<< HEAD
```

Comment:
what is this? you pushed changes without resolving the conflicts?

Reply:
it didn't show the diff in vscode for some reason. Anyway closing this PR because it's not needed.